src/views/CardStack/CardStackStyleInterpolator.js

```diff
@@ -68,17 +95,33 @@ function forHorizontal(
       index + 1,
     ]: Array<number>),
     outputRange: ([0, 1, 1, 0.85, 0]: Array<number>),
-  });
+  };
 
-  const translateY = 0;
-  const translateX = position.interpolate({
+  const translateX = {
     inputRange,
     outputRange,
-  });
+  };
+
+  if (isBack) {
+    if (index === lastIndex) {
+      translateX.inputRange[0] = activeIndex;
+      return {
+        transform: [{ translateX: position.interpolate(translateX) }],
+      };
+    } else if (index === activeIndex) {
+      translateX.inputRange[2] = lastIndex;
```

Comment:
Can you refactor this to use fewer magic numbers?

Reply:
According to the original animation effect, there is no way to reduce the magic numbers.